src/content/getting-started/observability/logging/audit-logs/index.md

```diff
@@ -0,0 +1,297 @@
+
+### Machine audit logs
+
+To provide you with a better visibility and security, Giant Swarm offers you the possibility to access the Machine audit logs.
+
+This solution relies on the [Linux Audit Daemon](https://linux.die.net/man/8/auditd) to collect the logs of any execution happening on the machine.
+It is currently configured as such:
+
+```
+-a exit,always -F arch=b64 -S execve -k auditing # -k auditing gives those logs a key so we can easily retrieve them
+-a exit,always -F arch=b32 -S execve -k auditing
+```
+
+This configuration asks the Linux Audit Daemon to trace any `execve` signal happening on the node and renders logs like:
+
+```sh
+Feb 06 09:57:37 ip-10-0-5-34.eu-central-1.compute.internal kernel: audit: type=1300 audit(1675677457.491:6132447): arch=c000003e syscall=59 success=yes exit=0 a0=55a8f8fc7fd0 a1=55a8f947eec0 a2=55a8f9b8fd10 a3=55a8f947eec0 items=2 ppid=1 pid=3659596 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="update-ssh-keys" exe="/usr/bin/update-ssh-keys" subj=system_u:system_r:kernel_t:s0 key="auditing"
+```
+
+Beware that the Linux Audit Daemon is quite verbose when it comes to logging so you will need to ensure enough storage and bandwidth for your logs. For this we suggest you use either a SIEM tool, Opensearch or Loki
+
+At the moment, the Machine audit logs can be found in `journald` so you can ship them over to your <abbr title="Security information and event management">SIEM</abbr> system.
+
+__Notice:__ The Linux Audit Daemon logs are kernel logs so they need to be accessed as such (e.g. `journalctl -kauditing`)
+
+The Machine audit logs will be shipped to our [Managed Loki](https://github.com/giantswarm/roadmap/issues/311) in the future.
+
+## How to ship your audit logs to a remote location
+
+Giant Swarm offers two tools to allow you to get the audit logs for your workload clusters.
+
+__Note__: You can also access the access logs for the management clusters but this is not something we provide by default (at least for now). Please contact your Account Engineer to sort out the details.
+
+### Fluent bit
+
+Giant Swarm provides a custom [Fluent-bit](https://fluentbit.io/) configuration packaged as a Managed App named [`fluent-logshipping-app`](https://github.com/giantswarm/fluent-logshipping-app)
+
+Fluent-bit's main advantage is that it can be used for all kinds of cases (shipping to rsyslog, elasticsearch, loki and so on) and provides a lot of configurability with it's plugin system so you can use it if this is what you need.
+
+### Promtail
+
+As we are providing [Loki as Managed app](https://github.com/giantswarm/loki-app), we also provide our customers with [Promtail](https://github.com/giantswarm/promtail-app) for log collection and shipping.
+
+Note that Promtail can only be used with Loki but it provides a better integration with it (targets and labels found using Service Discovery instead of on the node directly).
```

Comment:
I would keep it simple and focused on Promtail here

```suggestion
[Promtail](https://github.com/giantswarm/promtail-app) can only be used with Loki but it provides a better integration via its discovery mechanism rather than hardcoding file path.
```

Is this what you meant there? Also is it true that we can discover audits logs?

Reply:
No for audit logs we need to configure the path directly but for container logs that is true. @hervenicol can you confirm or not? :D